packages/admin/cms-admin/src/pages/useSaveConflictQuery.tsx

```diff
@@ -11,6 +11,7 @@ interface SaveConflictDialogOptions {
     hasChanges: boolean;
     loadLatestVersion: () => Promise<void>;
     onDiscardButtonPressed: () => Promise<void>;
+    pageTreeNodeId?: string;
```

Comment:
It's not nice to have a `pageTreeNodeId` here because useSaveConflictQuery() can also be used by non-documents (although `pageTreeNodeId` is optional).

Alternatively, we could pass a method that returns the required information (user and email). Downside: more configuration necessary in app